config/monica.php

```diff
@@ -12,6 +12,16 @@
     */
     'disable_signup' => env('APP_DISABLE_SIGNUP', false),
 
+   /*
+    |--------------------------------------------------------------------------
+    | Disable User registration
```

Comment:
Isn't it `Activate double optin on signup` instead?

Reply:
OK